src/rest.cpp

```diff
@@ -436,10 +463,13 @@ static bool rest_block(const std::any& context,
     }
 
     case RESTResponseFormat::JSON: {
+        if (!tx_verbosity.has_value()) {
```

Comment:
we should invert this to be able to use the otional in the idiomatic way of `if (opt) ... *opt`